ui.R

```diff
@@ -1,4 +1,9 @@
 
+variables <- c( "R1","R2","R1C","WO1","I")
+variables1 <- c("A1","A2","A1B","CM5","CM5B")
+variables2 <- c("CM1","CM2","CM4")
+variables3 <- c("I1","I9")
+variables4 <- c("RC1","RC4","RC9")
```

Comment:
There are some bypass meters rate code missing but I confirmed that we do not need to worry about them as it should not affect the rate model. As that information goes more into billing. I will follow your suggestion for setting up the rate code checkboxes.